pkg/drivers/kic/oci/errors.go

```diff
@@ -20,3 +20,6 @@ import "errors"
 
 // ErrWindowsContainers is thrown when docker been configured to run windows containers instead of Linux
 var ErrWindowsContainers = errors.New("docker container type is windows")
+
+// ErrCPUCountLimit is thrown when docker daemon doesn't have enough CPUs for the requested container
+var ErrCPUCountLimit = errors.New("not enough CPUs is available for container")
```

Comment:
You could create a `FailFastError` type to wrap around these errors, so that we don't need to check every type of error in `node/start.go`

```
type FailFastError error
var ErrWindowsContainers = FailFastError(errors.New("docker container type is windows"))
var ErrCPUCountLimit = FailFastError(errors.New("not enough CPUs is available for container"))
```

Reply:
Nice tip!

Reply:
done